gameview.py

```diff
@@ -320,6 +323,7 @@ def check_collisions(self):
 
                     # Play enemy death sfx
                     Audio.play_rand_sound(enemy.sfx_death_list)
+                    self.enemy_killed += 1  # Increment Enemy killed
```

Comment:
I don't think we want the shop to open automatically. I would remove this and just use the one inside map menu.

Reply:
also lines 66 - 68